erpnext_com/www/docs/user/manual/en/CRM/appointment.md

```diff
@@ -0,0 +1,58 @@
+# Appointment
```

Comment:
please crop the top part in images to remove the URL bar. In the third image retake, the developer field is visible, also ensure to take screenshot without interference in the top. 

List the files in index.md.